### PR TITLE
Add user memberships

### PR DESCRIPTION
**Pull request by lucasmichot**
Add user memberships features:
- https://developer.github.com/v3/orgs/members/#list-your-organization-memberships
- https://developer.github.com/v3/orgs/members/#get-your-organization-membership
- https://developer.github.com/v3/orgs/members/#edit-your-organization-membership

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Please indent with spaces. :P

**Comment by lucasmichot** [comment]
good catch Graham

**Comment by cursedcoder** [comment]
very good @lucasmichot do you want to add docs here either?

**Comment by lucasmichot** [comment]
Yep @cursedcoder I will do so, as for orgs webhooks

**Comment by lucasmichot** [comment]
Ping @cursedcoder

**Comment by GrahamCampbell** [comment]
@lucasmichot You need to rebase this to load the correct styleci config.

**Comment by lucasmichot** [comment]
Yep saw that done @GrahamCampbell

**Comment by GrahamCampbell** [comment]
:)

**Comment by lucasmichot** [comment]
Nice https://styleci.io/commits/648bc8614e77894532343fa4d0d762e77886af3f :+1:

File: lib/Github/Api/CurrentUser/Memberships.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    /**
+     * Edit your organization membership
+     *
+     * @link https://developer.github.com/v3/orgs/members/#edit-your-organization-membership
+     *
+     * @return array
+     */
+    public function edit($organization)
```
Comment (stof) [inline lib/Github/Api/CurrentUser/Memberships.php]: the name is weird, because this method only allows to mark the membership as active.

**Pull request merged**